sdk/nextjs/src/server/setupDevCycle.tsx

```diff
@@ -70,6 +100,19 @@ export const setupDevCycle = <
         return client.allFeatures()
     }
 
+    const _track = async (event: DevCycleEvent) => {
+        if (!options.disableCustomEventLogging) {
```

Comment:
prefer using the early return instead of wrapping this the whole function in a NOT condition